### PR TITLE
Add DIAData Space

### PR DESCRIPTION
**Pull request by bitsandtea**
Etherscan: https://etherscan.io/token/0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419
CoinGecko: https://www.coingecko.com/en/coins/dia
Website: https://diadata.org
Twitter: https://twitter.com/DIAdata_org

Great tool, looking forward to our first vote! We might change the skins later. Thanks

### CODE REVIEW

File: skins/diadata.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.diadata {
+    --primary-color: #4F33FF;
+    --bg-color: CCE0F4;
```
Comment (gokaiorg) [inline skins/diadata.scss:3]: You miss a #. 
```suggestion
    --bg-color: #CCE0F4;
```

File: spaces/diadata/index.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "token": "0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419",
+  "name": "DIAData",
+  "network": "1",
+  "symbol": "DIA",
+  "domain": "vote.diadata.org",
```
Comment (gokaiorg) [inline spaces/diadata/index.json:6]: You will need `skin` field to add your custom styles.
```suggestion
  "skin": "diadata",
  "domain": "vote.diadata.org",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/diadata/index.json (hunk 2 above)
Comment (gokaiorg) [reply spaces/diadata/index.json:6]: To add your custom domain you will have to add your domain in `spaces/domains.json` file also.
https://docs.snapshot.page/guides/custom-domain

Comment (bitsandtea) [reply spaces/diadata/index.json:6]: Added, thanks

File: skins/diadata.scss (hunk 1 above)
Comment (bitsandtea) [reply skins/diadata.scss]: Added, thanks

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
The file `spaces/domains.json` has been updated since your last commit, you should checkout this file and commit again your changes. Thank you.

**Comment by bitsandtea** [comment]
> The file `spaces/domains.json` has been updated since your last commit, you should checkout this file and commit again your changes. Thank you.

just re-committed it!

**Pull request merged**